### PR TITLE
added docker image for ECS, added docs

### PR DESCRIPTION
**Pull request by ViacheslavSaievskyi**
Docker file to building docker image was added as an example of how it could work with AWS ECS
Entered minimal documentation

### CODE REVIEW

File: elastio-stream-kafka/README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
 # Elastio Stream (MSK)Kafka <!--Discuss another naming-->
+
 # Quick start
-## Create machine:
-1. Create an EC2 instance( Amazon Linux 2 AMI (HVM) - Kernel 5.10, SSD Volume Type, instance type: t2.xlagre) in the same VPC as your MSK cluster.
-2. After the EC2 instance was created copy the name of the security group, and save it for later. 
+
+## Create an instance:
+1. Create an EC2 instance (Amazon Linux 2 AMI (HVM) - Kernel 5.10, SSD Volume Type, instance type: t2.xlagre) in the same VPC as your MSK cluster.
```
Comment (swarmer) [inline elastio-stream-kafka/README.md:6]:
```suggestion
1. Create an EC2 instance (Amazon Linux 2 AMI (HVM) - Kernel 5.10, SSD Volume Type, instance type: t2.xlarge) in the same VPC as your MSK cluster.
```

File: elastio-stream-kafka/README.md [hunk 2]
```diff
@@ -57,4 +59,69 @@
...
+    2. In the navigation tab choose `Scheduled Tasks`.
+    3. Press `Create` button.
+    4. Enter `Schedule rule name`=>`Target id`.
+    5. Launch type choose `Fargate`.
+    6. Task Definition, choose your task definition.
+    > If you want to backup a topic from the AWS MSK, select the same VPC as you selected for MSK. Task Security Group has to accept Kafka traffic.
+    7. Select a VPC from the list in the Cluster VPC field.
+    8. Add at least one subnet.
+    9. Enable Auto-assign public IP.
+    10. Press `Create` button. This will run the task according to the selected schedule.
```
Comment (swarmer) [inline elastio-stream-kafka/README.md:127]: Will this push logs anywhere? I feel like this needs a follow-up section describing how to see the logs for troubleshooting.

File: elastio-stream-kafka/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+RUN apt-get install -y python3-pip python3
+COPY requirements.txt  ./
+RUN python3 -m pip install --no-cache-dir -r requirements.txt
+COPY common.py ./
+COPY elastio_stream_kafka.py ./
+COPY docker_backup_app.py ./
+COPY docker_backup_config.py ./
+
+ENTRYPOINT [ ]
+CMD python3 docker_backup_app.py
```
Comment (ivankovnatsky) [inline elastio-stream-kafka/Dockerfile:19]:
```suggestion
ENTRYPOINT ["python3"]
CMD ["docker_backup_app.py"]
```

File: elastio-stream-kafka/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1
+
+# Version elastio cli what will use like base image.
+# Most likely this is not the latest version of elastio cli.
+# You can find the latest version at this link.
+# https://gallery.ecr.aws/elastio-dev/elastio-cli
+FROM public.ecr.aws/elastio-dev/elastio-cli:0.17.0-master
+
+RUN apt-get update && apt-get upgrade -y
```
Comment (ivankovnatsky) [inline elastio-stream-kafka/Dockerfile:9]:
```suggestion
RUN apt-get update && apt-get upgrade -y --no-install-recommends --no-install-suggests
```

Comment (ivankovnatsky) [inline elastio-stream-kafka/Dockerfile:9]: I would not upgrade, rather just taking a needed base image with newest relevant tag.

File: elastio-stream-kafka/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+RUN apt-get install -y python3-pip python3
```
Comment (ivankovnatsky) [inline elastio-stream-kafka/Dockerfile:10]: Why not put apt-get install as a single RUN?

**Review by ivankovnatsky — CHANGES_REQUESTED** [review]

File: elastio-stream-kafka/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# syntax=docker/dockerfile:1
+
+# Version elastio cli what will use like base image.
+# Most likely this is not the latest version of elastio cli.
+# You can find the latest version at this link.
+# https://gallery.ecr.aws/elastio-dev/elastio-cli
+FROM public.ecr.aws/elastio-dev/elastio-cli:0.17.0-master
```
Comment (ivankovnatsky) [inline elastio-stream-kafka/Dockerfile:7]: Per Adam let's use `:latest` here.

**Pull request closed without merging**